### ISSUE TITLE
[TLD] Report next red light without actual detection

### ISSUE THREAD

**Issue by jodiealaine** [clockworkbits/CarND-Capstone#6]
**Scope:** 
We need a way to read the state of all traffic lights from the /vehicle/traffic_lights topic, in order to identify which traffic light is next on the car's path and to find out if it is red or not.

**Out of Scope:** 
Does not need to detect the color of the light (which is the real work of tl_detector); this is just to unblock work on the waypoint updater

**Comment by jodiealaine** [comment]
Completed by @agotterba in the following PR: [Report next red light without actual detection](https://github.com/clockworkbits/CarND-Capstone/pull/5)
